test/runner/lib/core_ci.py

```diff
@@ -238,7 +239,7 @@ def get(self, tries=2, sleep=10, always_raise_on=None):
 
         status = 'running' if self.connection.running else 'starting'
 
-        display.info('Retrieved %s %s/%s instance %s.' % (status, self.platform, self.version, self.instance_id),
+        display.info('Polling when %s %s/%s on instance %s...' % (status, self.platform, self.version, self.instance_id),
```

Comment:
Since this is displayed after the polling has occurred, I think the new messaging isn't any more clear. How's this?

```python
display.info('Retrieved %s/%s instance %s (status: %s)' % (self.platform, self.version, self.instance_id, status),
```